easybuild/easyconfigs/s/Siesta/Siesta-4.0-foss-2017b.eb

```diff
@@ -0,0 +1,18 @@
+name = 'Siesta'
+version = '4.0'
+
+homepage = 'http://departments.icmab.es/leem/siesta'
+description = """SIESTA is both a method and its computer program implementation, to perform efficient electronic
+ structure calculations and ab initio molecular dynamics simulations of molecules and solids."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'usempi': True}
+
+sources = [SOURCELOWER_TGZ]
+vl = version.split('-')
+source_urls = ['https://launchpad.net/siesta/%s/%%(version)s/+download' % vl[0]]
+checksums = ['c4aa5f4cf2eabefaaacd3ef6350ec8fd']
```

Comment:
can you add a `sha256` checksum instead? (`--inject-checksums`, since easybuild `3.5.0`, will do it for you)